vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -120,7 +120,7 @@
       <PrecompiledHeaderOutputFile>$(IntDir)pch.pch</PrecompiledHeaderOutputFile>
       <ForcedIncludeFiles>pch.h</ForcedIncludeFiles>
       <WarningLevel>Level4</WarningLevel>
-      <AdditionalOptions>/await %(AdditionalOptions) /bigobj</AdditionalOptions>
+      <AdditionalOptions>/await %(AdditionalOptions) /bigobj /showIncludes</AdditionalOptions>
```

Comment:
>showIncludes [](start = 62, length = 12)

Is this intentional? Or just here from diagnostics?

Reply:
This was an oversight, I removed in the latest iteration of the change.

Reply:
@dannyvv @aeulitz  FYI here is a nice way to preprocess C++ files (while respecting include paths etc.)

https://gist.github.com/asklar/f94fc2d1867cefe0f5d3c70d59c8ccba

Reply:
Thanks for the tip, Alex.

---
In reply to: [427003089](https://github.com/microsoft/react-native-windows/pull/4924#discussion_r427003089) [](ancestors = 427003089)

Reply:
I removed this in the later iterations of the change. Thanks for the catch!

---
In reply to: [426967955](https://github.com/microsoft/react-native-windows/pull/4924#discussion_r426967955) [](ancestors = 426967955)